fast/stages/1-resman/variables.tf

```diff
@@ -189,6 +183,18 @@ variable "locations" {
   nullable = false
 }
 
+variable "org_policy_tags" {
```

Comment:
should this be `org_policies_tags`? I'm not sure.

Reply:
I'm never sure about those things, let's look at an English language rules reference once and for all...

Reply:
seems like the current version is ok

https://grammar.collinsdictionary.com/easy-learning/how-do-you-form-the-plural-of-compound-nouns-in-english#:~:text=Grammar%20%7C%20Collins%20Education-,How%20do%20you%20form%20the%20plural%20of%20compound%20nouns%20in,last%20word%20of%20the%20compound.